### ISSUE TITLE
Week 3 Convert Uno model to javascript - Kevin

### ISSUE THREAD

**Issue by keviin0** [sgtran/p1---more-rice#25]
Convert components so we don't have to rely on replit
- [x] Create initial html components in controller for each player move.
- [x] Convert game logic into javascript/Spring
- [x] Convert control into javascript (initialization)

**Comment by Dragon0344** [comment]
Maybe, just leave it as java and code the java into html.(I'm fairly certain there's methods)

**Comment by domph** [comment]
Might want to clarify what kind of game logic is being converted on the ticket - is it everything? a certain component? etc.

Possible suggestions:
- Could go the route where the Swing program is simply run in a web-format
- JS may be necessary for more dynamic interaction since Java is all back-end (JS could encapsulate all the graphics, or maybe some HTML/CSS parts as well?)

Overall, it'll be tough since it requires JS knowledge, but I think it'll be a fun challenge.

**Comment by DaHippoBear** [comment]
Sounds like a good plan, you might want to include that you are going to leave some things in Java and use Spring to display, just so you have it on paper as well. Also explain why you would want to use javascript as well. This is definitely a project that has been thought out, and I look forward to seeing it done!

**Comment by keviin0** [comment]
1 and 3 added in 15af77277561ae985bdf4acae8b8fe6e283e67c2 and e498aa9a7cbfe055ed874f2c3999d6f31891c179.

**Comment by keviin0** [comment]
2 completed in 1f9547db0217a2dd8d35857b90f4d49f2cd08a66.

**Comment by sgtran** [comment]
*Make sure to use different branch while working on this to ensure rest team can work on main
*Keep working hard. We are making good progress.

**Comment by AlexHhhhhhh** [comment]
Same as above, unless only changing controller and not affecting anything else (?)

**Comment by andrewzpu** [comment]
Looks like a lot of good code to switch to the new version of Uno. As Sean said, make sure to use a different branch so that we can still work.

**Comment by athkud123** [comment]
Good idea to use spring for uno, make sure this is visible on website. good that you provided links to code.
